server/routes/notifications/createNotification.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express')
+const router = express.Router()
+const ajv = require('../../schema')
+const { createNotificationSchema } = require('../../schema/notifications')
+const middleware = require('../middlewares')
+const notifications = require('../../models/notifications')
+
+/**
+ *
+ * @param {Array} errArray
+ * @return {String}
+ */
+function sumErrors(errArray) {
+  const cb = (a, b) => a + b.message + ', '
+  return errArray.reduce(cb, '')
+}
+
+router.post(
+  '/',
+  middleware.verifyUserAccessToken,
+  middleware.verifyAdmin,
```

Comment:
No need for `verifyAdmin` Moderator can also add members which will create notifications. The only thing is we need to check if the user is moderator.

Reply:
No, that would be an automatic notifiaction

Reply:
i see...